tests/src/python/test_provider_postgres.py

```diff
@@ -554,7 +554,7 @@ def testGeneratedFields(self):
         expected_area = 43069568296.34387
 
         assert compareWkt(generated_geometry, expected_geometry), "Geometry mismatch! Expected:\n{}\nGot:\n{}\n".format(expected_geometry, generated_geometry)
-        self.assertEqual(f2['poly_area'], expected_area)
+        self.assertEqual(round(f2['poly_area'], 4), round(expected_area, 4))
```

Comment:
You can use self.assertAlmostEqual here instead, which will give nicer debug messages when the test fails

Reply:
I'll try this, thanks for the pointer!

Reply:
Fixed by 9537f48